avocado/utils/unit_converter.py

```diff
@@ -0,0 +1,46 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
Although this is probably better location, there are other converting utils in `avocado.utils.data_structures`. I think we should merge them. It'd be quite problematic to move existing functions, so I'd suggest adding this to the `data_structures`.